webkit2gtk-sys/tests/layout.c

```diff
@@ -1,5 +1,5 @@
 // This file was generated by gir (https://github.com/gtk-rs/gir)
-// from gir-files (https://github.com/gtk-rs/gir-files)
```

Comment:
That doesn't look right, you forgot to update the gir_directories here as well i suppose?

Reply:
Not sure what happened there, resolved now